### PR TITLE
AB#9260 Create Products (5-col promo) component

### PR DESCRIPTION
**Pull request by chpa7**

### CODE REVIEW

File: src/items/sxa-content/content/Verticals/Financial/Settings/Site Grouping/Financial.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ SharedFields:
   Value: Financial
 - ID: "8e0dd914-9afb-4d45-bf8b-7ff5d6e5337e"
   Hint: HostName
-  Value: "www.sxastarter.localhost|localhost|verticals-development-financial.vercel.app"
+  Value: "chpa-test-5-col-nf1qhuqnqks5t3n7jr3eaw-financial.vercel.app"
```
Comment (adoprog) [inline src/items/sxa-content/content/Verticals/Financial/Settings/Site Grouping/Financial.yml:18]: Please revert the hostname change

File: src/items/sxa-content/content/Verticals/Basic/Settings/Site Grouping/Basic.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ SharedFields:
   Value: Basic
 - ID: "8e0dd914-9afb-4d45-bf8b-7ff5d6e5337e"
   Hint: HostName
-  Value: basic.sxastarter.localhost
+  Value: "chpa-test-5-col-nf1qhuqnqks5t3n7jr3eaw-basic.vercel.app"
```
Comment (adoprog) [inline src/items/sxa-content/content/Verticals/Basic/Settings/Site Grouping/Basic.yml:18]: Please revert the hostname change

File: src/items/sxa-content/content/Verticals/Services/Settings/Site Grouping/Services.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ SharedFields:
   Value: Services
 - ID: "8e0dd914-9afb-4d45-bf8b-7ff5d6e5337e"
   Hint: HostName
-  Value: services.sxastarter.localhost
+  Value: "chpa-test-5-col-nf1qhuqnqks5t3n7jr3eaw-services.vercel.app"
```
Comment (adoprog) [inline src/items/sxa-content/content/Verticals/Services/Settings/Site Grouping/Services.yml:18]: Please revert the hostname change

File: src/items/sxa-content/content/Verticals/Financial/Home/finance-for-you/Data/Text 3.yml [hunk 4]
```diff
@@ -7,15 +7,3 @@ Languages:
 - Language: en
   Versions:
   - Version: 1
-    Fields:
```
Comment (adoprog) [inline src/items/sxa-content/content/Verticals/Financial/Home/finance-for-you/Data/Text 3.yml:10]: Let's keep the sample text while we don't have real content.

**Review by adoprog — CHANGES_REQUESTED** [review]

File: src/components/collections.json [hunk 5]
```diff
@@ -0,0 +1,127 @@
+[
```
Comment (adoprog) [inline src/components/collections.json:1]: Ok to add Components, but let's not push them as changed every time, unless there is a change to actual components content.

File: src/items/sxa-content/content/Verticals/Financial/Home/Data/Five Column CTA 2.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿---
```
Comment (adoprog) [inline src/items/sxa-content/content/Verticals/Financial/Home/Data/Five Column CTA 2.yml:1]: Do we need all these three items? Let's leave the one that is used by the rendering we have at home page.

**Comment by chpa7** [comment]
Reverted the "Demo Portal - automated items sync" commits.

**Review by adoprog — APPROVED** [review]

**Pull request merged**